siriuspy/siriuspy/devices/ids.py

```diff
@@ -1717,6 +1717,55 @@ def cmd_reset(self, timeout=None):
             self.PARAM_PVS.RESET, timeout=timeout)
 
 
+class VPU(APU):
+    """VPU Insertion Device."""
+
+    class DEVICES:
+        """Device names."""
+
+        VPU29_06SB = 'SI-06SB:ID-VPU29'
+        # VPU29_07SP = 'SI
+        # -07SP:ID-VPU29'
+        ALL = (VPU29_06SB, )  # VPU29_07SP, )
+
+    # DevCtrl PV
+    # _CMD_MOVE_STOP, _CMD_MOVE_START = 300, 320
+    # _CMD_RESET, _CMD_SCAN_START = 310, 340
+    # _CMD_SCAN_MODE = 330
+
+    PARAM_PVS = _ParamPVs()
+
+    # --- GENERAL ---
+    PARAM_PVS.PERIOD_LEN_CTE = 'PeriodLength-Cte'
+    PARAM_PVS.START_PARKING_CMD = 'StartParking-Cmd'
+    PARAM_PVS.IS_MOVING = 'Moving-Mon'
+    PARAM_PVS.MOVE_ABORT = 'Abort-Cmd'
+    # PARAM_PVS.RESET = None
+
+    # --- KPARAM ---
+    PARAM_PVS.KPARAM_SP = 'KParam-SP'
+    PARAM_PVS.KPARAM_RB = 'KParam-SP'  # Can we read from CPL?
+    PARAM_PVS.KPARAM_MON = 'KParam-Mon'
+    PARAM_PVS.KPARAM_PARKED_CTE = 'KParamParked-Cte'
+    # PARAM_PVS.KPARAM_MAXACC_SP = None
+    # PARAM_PVS.KPARAM_MAXACC_RB = None
+    PARAM_PVS.KPARAM_MAXVELO_SP = 'KParamMaxVelo-SP'
+    PARAM_PVS.KPARAM_MAXVELO_RB = 'KParamMaxVelo-RB'
+    PARAM_PVS.KPARAM_VELO_SP = 'KParamVelo-SP'
+    PARAM_PVS.KPARAM_VELO_RB = 'KParamVelo-RB'
```

Comment:
there is no -RB here from CLP, so the correct here is `PARAM_PVS.KPARAM_VELO_RB = 'KParamVelo-SP'`